wgsl/index.bs

```diff
@@ -2735,7 +2734,8 @@ statement
 A function declaration may only occur at module scope.
 The function name is available for use after its declaration, until the end of the program.
 
-Functions must end with a `return` statement. The return may be given with a
+Functions returning a value must end with a `return` statement. Functions returning
```

Comment:
This might be taken as circular.   I'd suggest phrasing it in a parallel fashion to the statement above:

> If the return type of the function is not the void type, then the last statement in the function body must be a return statement.

Note that the [return statement section](https://gpuweb.github.io/gpuweb/wgsl.html#return-statement) already talks about type-matching to the function return type.  So i think we should remove the "The return may be given with a value to be returned." sentence.

Reply:
Done